app.js

```diff
@@ -14,18 +14,44 @@ Words only consist of letters, never apostrophes or other punctuation symbols.
   - identify most frequent word and return 
   */
 
- var nDictionary = Object.create(null);
+ function getWordsWithHighestFreq(paragraphArr) { 
+    var frequencyDict = new Object(); 
 
- function setDictionary(index, value) {
-     nDictionary[index] = value;
- }
- 
- function getDictionary(index) {
-     return nDictionary[index];
- }
+    //Adds words from paragraph into a dictionary 
+    for (var i = 0; i < paragraphArr.length; i++) {
+      var key = paragraphArr[i]; 
 
- getDictWithFrequencies(paragraphArr, bannedWordsArr) { 
-    var frequencyDict = new Object(); 
+      if(key in frequencyDict) {
+        value = frequencyDict[key]; 
```

Comment:
I'm pretty sure you should have declare 'value' with var or let here, unlike Object.entries which provides [key, value].
Without a var, it becomes an implicit global level variable

Reply:
pretty sure just in case I'm missing something like Object.entries